### PR TITLE
string condition added

### PR DESCRIPTION
**Pull request by sanjeevprabhakar**

### CODE REVIEW

File: whileloopfactorial.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function factfunction(a){
```
Comment (darshansharma) [inline whileloopfactorial.js]: wrong function name

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: buttoncount.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
-function(){
+function fun(){
```
Comment (darshansharma) [inline buttoncount.js]: This function is not indentated properly. 
Why so much extra empty lines. 
Don't direct copy,paste from internet, atleast after getting code from internet, type it yourself

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: button.html [hunk 3]
```diff
@@ -7,7 +7,9 @@
 </head>
 
 <body>
```
Comment (darshansharma) [inline button.html:10]: Why script tag is present inside body tag ?

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: buttoncount.js [hunk 4]
```diff
@@ -1,12 +1,8 @@
-function fun(){
+function fun() {
```
Comment (darshansharma) [inline buttoncount.js:2]: who so many empty lines are present ??
Use eslint in vs code = Ctrl + shift + I

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: buttoncount.js [hunk 5]
```diff
@@ -1,8 +1,5 @@
-function fun() {
-
+function fun(){
     click = 0
```
Comment (darshansharma) [inline buttoncount.js]: Where is the declaration of variable??

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: button.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <meta charset="utf-8">
+    <title> button magic </title>
+    <script> src = "./buttoncount.js"</script>
+</head>
+
+<body>
+    
+    <button onclick="fun()" > click me </button>
+    <div> clicks: <a id ="count"> 0 </a></div>
```
Comment (darshansharma) [inline button.html]: Why have you used <a> anchor  tag here? What is the reason behind it?

**Review by darshansharma — CHANGES_REQUESTED** [review]

File: button.html (hunk 3 above)
Comment (sanjeevprabhakar) [reply button.html:10]: changed the  script tag location to the head tag.

File: buttoncount.js (hunk 5 above)
Comment (sanjeevprabhakar) [reply buttoncount.js]: silly mistake i resolve.

File: button.html (hunk 6 above)
Comment (sanjeevprabhakar) [reply button.html]: i removed the anchor tag .

**Pull request merged**